.github/workflows/common-ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Common CI
+
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
I'm not fully sure about this paths, can you double check @korthout ? I think we should ignore everything related to Zeebe, as operate doesn't have a CI in 8.5